install.sh

```diff
@@ -114,7 +114,9 @@ check() {
                 echo >> "$shellrc"
             fi
             echo "${2:-$path}" >> "$shellrc"
-            log "Restart your shell to have spicetify in your PATH."
+            # Source the rc file to have spicetify binary available already in the session
+            # shellcheck source=/dev/null
```

Comment:
```suggestion
            # shellcheck disable=SC1090
```

Reply:
the SC1090 mentions that the `shellcheck source=/dev/null` disables it